docs/src/explanations/radial-integral.rst

```diff
@@ -0,0 +1,278 @@
+.. _radial-integral:
```

Comment:
I like having this page around, but it does not flow very well yet (in  part because it assumes knowledge of spherical expansion & friends, which should be explained in a different section). We can still merge it now and improve it later!

Reply:
Might be worth to move it to some "advanced documentation" section

Reply:
Yes, fine for me.

Reply:
I moved it to the devdocs. I think there it make more sense in its current state. I also added a sentence:


> Note that this page assumes knowledge of spherical expansion & friends and currently serves as
> a reference page for the developers to support the implementation.